boa/core/build.py

```diff
@@ -511,43 +513,54 @@ def download_source(m):
 
 
 def build(m, stats=None):
-    if not stats:
-        stats = {}
+    try:
+        if not stats:
+            stats = {}
 
-    if m.skip():
-        print(utils.get_skip_message(m))
-        return {}
+        if m.skip():
+            print(utils.get_skip_message(m))
+            return {}
 
-    with utils.path_prepended(m.config.build_prefix):
-        env = environ.get_dict(m=m)
+        with utils.path_prepended(m.config.build_prefix):
+            env = environ.get_dict(m=m)
 
-    env["CONDA_BUILD_STATE"] = "BUILD"
-    if env_path_backup_var_exists:
-        env["CONDA_PATH_BACKUP"] = os.environ["CONDA_PATH_BACKUP"]
+        env["CONDA_BUILD_STATE"] = "BUILD"
+        if env_path_backup_var_exists:
+            env["CONDA_PATH_BACKUP"] = os.environ["CONDA_PATH_BACKUP"]
 
-    m.output.sections["package"]["name"] = m.output.name
-    env["PKG_NAME"] = m.get_value("package/name")
+        m.output.sections["package"]["name"] = m.output.name
+        env["PKG_NAME"] = m.get_value("package/name")
 
-    src_dir = m.config.work_dir
-    if isdir(src_dir):
-        if m.config.verbose:
-            print("source tree in:", src_dir)
-    else:
-        if m.config.verbose:
-            print("no source - creating empty work folder")
-        os.makedirs(src_dir)
+        src_dir = m.config.work_dir
+        if isdir(src_dir):
+            if m.config.verbose:
+                print("source tree in:", src_dir)
+        else:
+            if m.config.verbose:
+                print("no source - creating empty work folder")
+            os.makedirs(src_dir)
 
-    utils.rm_rf(m.config.info_dir)
-    files_before_script = utils.prefix_files(prefix=m.config.host_prefix)
+        utils.rm_rf(m.config.info_dir)
+        files_before_script = utils.prefix_files(prefix=m.config.host_prefix)
 
-    with open(join(m.config.build_folder, "prefix_files.txt"), "w") as f:
-        f.write("\n".join(sorted(list(files_before_script))))
-        f.write("\n")
+        with open(join(m.config.build_folder, "prefix_files.txt"), "w") as f:
+            f.write("\n".join(sorted(list(files_before_script))))
+            f.write("\n")
 
-    execute_build_script(m, src_dir, env)
+        execute_build_script(m, src_dir, env)
 
-    if m.output.sections["build"].get("intermediate"):
-        utils.rm_rf(m.config.host_prefix)
-        return
+        if m.output.sections["build"].get("intermediate"):
+            utils.rm_rf(m.config.host_prefix)
+            return
+
+        bundle_conda(m, files_before_script, env, m.output.sections["files"])
+    except subprocess.CalledProcessError:
+        print(sys.exc_info()[1])
+        print(f"$PREFIX = {m.config.build_prefix}")
+
+        build_cmd = str(
+            pathlib.Path(m.config.build_prefix).parent / "work/conda_build.sh"
+        )
+        print(f"Try building again with {build_cmd}")
```

Comment:
looks good!
note there are some differences on windows (conda_build.bat vs conda_build.sh, and the direction of slashes).

Also you might want to try `console.print("[red]ERROR[/red]")` etc for fancy markup (we're using rich for that now).